chatbot-ui/templates/deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     spec:
       containers:
         - name: {{ .Values.name }}
-          image: {{ tpl .Values.image $ }}
+          image: {{ tpl chatbot-ui.imageref . }}
```

Comment:
Since this is using a helm helper file, lets swap this to `include chatbot-ui.imageref .`

tpl should work, but `include` is the more helm idiomatic way to do this and will help others to infer where to look for the function.